2.4/Dockerfile

```diff
@@ -36,6 +37,11 @@ RUN yum install -y epel-release && \
 COPY run-mongod.sh /usr/local/bin/
 COPY contrib /var/lib/mongodb/
 
+# Due to the bug https://bugzilla.redhat.com/show_bug.cgi?id=1206151, 
+# the .bashrc has to be copied to the wrong $HOME dir so when using
+# 'docker exec' the mongodb collection is enabled automatically.
```

Comment:
I'd make sure to mention that this applies to 'docker exec bash' because e.g. 'docker exec mongod' won't work.